Cython/Build/Dependencies.py

```diff
@@ -1034,7 +1034,7 @@ def copy_to_build_dir(filepath, root=os.getcwd()):
                 # setup for out of place build directory if enabled
                 if build_dir:
                     if os.path.isabs(c_file):
-                        warnings.warn("build_dir has no effect for absolute source paths")
+                        c_file = c_file.split(os.sep, 1)[1]
```

Comment:
I guess this is meant to turn an absolute path into a relative path with the same directory names. Can't say right now if this always works in a cross-platform way. What about Windows network drives, for example? (`\\server\path`)

Reply:
I have added `os.path.splitdrive()` to remove windows/network drive. This should cover all cases.